### ISSUE TITLE
Grace note documentation

### ISSUE THREAD

**Issue by davidrizo** [humdrum-tools/vhv-documentation#9]
Please, when possible it would be very useful to have documentation about grace notes. I've found the ones related to cue-sized notes, but not that for appoggiatura and acciaccatura. 

At least it would be nice to know what part of the explanation about it in https://www.humdrum.org/rep/kern/ is supported in VHV. I've tested the encoding of acciaccaturas with `q` and it works, but not the appoggiatura with `P` and `p`, or the grace note groups.

Thanks.

**Comment by craigsapp** [comment]
Yes, there are difference between the official documentation and implementation in VHV (which I will fix sometime...).

The short answer is that in VHV "q" is an acciaccatura, and "qq" is an appoggiatura, and "Q" is not used (but I may remap it to cue-sized notes).  "P" and "p" are qualifiers on appogiatura notes.

Here are some examples of each case:

![gracenotes](https://user-images.githubusercontent.com/3487289/79981433-8534b080-8459-11ea-86c8-680f74241136.png)

The first measure is an acciaccatura, the second is an appogiatura.  To be more precise, the "q" means that the note is a grace note, and that the grace note has a slash on it.  The "qq" marker means that the note is a grace note and it does not have a slash.  So "q" and "qq" are focused on notational aspects of the grace notes.  There are certain styles of music notation where acciaccaturas are given without a slash, so "qq" does not guarantee an appogiatura, only that "qq" likely means an appogiatura based on common graphical representation of appogiaturas.

To be more explicit for appogiaturas, a "P" should be added to the grace note as shown in measure 3, and then a "p" on the following note.  Measure 4 shows the grace note converted into a regular note in a performance rendering of the grace note.  The modern style is to display  most appogiaturas in measure 4 format, where the original 18th-century edition would have used the measure 3 format.

In other words "q" and "qq" are graphical descriptions, and "P"/"p" are analytic descriptions.

**Comment by mscuthbert** [comment]
May I ask that existing letters like Q (=qq) not be remapped to new letters at least so long as humdrum/kern remains versionless?  It is very hard to tell whether an encoding using Q means the older Gruppetto symbol or now the "cue sized note" it is apparently being used to mean now.  Thanks!

**Comment by craigsapp** [comment]
You can try your luck at MedRen in two weeks.

**Comment by mscuthbert** [comment]
> You can try your luck at MedRen in two weeks.

Looking forward to conversation...and maybe Perogis too!

**Comment by craigsapp** [comment]
NB: I may miss the first day, leaving July 5th, but I will be there the
rest of the month
